### PR TITLE
Detect .code-workspace.jinja files

### LINKED ISSUE #46: *.code-workspace.jinja files not detected as JSON
*.code-workspace.jinja files should be jinja-jsonc highlighted by default.

### PR DESCRIPTION
**Pull request by Yajo**
Fix #46.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -45,7 +45,7 @@
       {
         "id": "jinja-json",
         "aliases": ["Jinja JSON", "jinja-json"],
-        "extensions": [".json.j2", ".json.jinja", ".json.jinja2"],
+        "extensions": [".json.j2", ".json.jinja", ".json.jinja2", ".code-workspace.j2", ".code-workspace.jinja", ".code-workspace.jinja2"],
```
Comment (samuelcolvin) [inline package.json:48]: have you tried this? are we sure it will take priority over `jinja-html` with has `".jinja"` etc.?

Comment (Yajo) [reply package.json:48]: I haven't tried, but if it works for JSON, there should be no difference...

Comment (samuelcolvin) [reply package.json:48]: sorry, I'm not sure what you mean. Is this working?

Comment (Yajo) [reply package.json:48]: Yes.

Comment (Yajo) [reply package.json:48]: Well, really it should be affected by #73 also, but other than that it should work.

**Pull request merged**